docs/reST/ref/code_examples/draw_module_example.py

```diff
@@ -1,94 +1,92 @@
-    
-    # Draw on the screen a GREEN line from (0, 50) to (50, 80) 
+    screen.fill("white")
+
+    # Draw on the screen a green line from (0, 0) to (50, 30)
+    # 5 pixels wide. Uses (r, g, b) color - medium sea green.
+    pygame.draw.line(screen, (60, 179, 113), [0, 0], [50, 30], 5)
+
+    # Draw on the screen a green line from (0, 50) to (50, 80)
     # Because it is an antialiased line, it is 1 pixel wide.
-    pygame.draw.aaline(screen, GREEN, [0, 50],[50, 80], True)
+    # Uses (r, g, b) color - medium sea green.
+    pygame.draw.aaline(screen, (60, 179, 113), [0, 50], [50, 80], True)
+
+    # Draw on the screen 3 black lines, each 5 pixels wide.
+    # The 'False' means the first and last points are not connected.
+    pygame.draw.lines(
+        screen, "black", False, [[0, 80], [50, 90], [200, 80], [220, 30]], 5
+    )
 
     # Draw a rectangle outline
-    pygame.draw.rect(screen, BLACK, [75, 10, 50, 20], 2)
-     
-    # Draw a solid rectangle
-    pygame.draw.rect(screen, BLACK, [150, 10, 50, 20])
+    pygame.draw.rect(screen, "black", [75, 10, 50, 20], 2)
+
+    # Draw a solid rectangle. Same color as "black" above, specified in a new way
+    pygame.draw.rect(screen, (0, 0, 0), [150, 10, 50, 20])
 
     # Draw a rectangle with rounded corners
-    pygame.draw.rect(screen, GREEN, [115, 210, 70, 40], 10, border_radius=15)
-    pygame.draw.rect(screen, RED, [135, 260, 50, 30], 0, border_radius=10, border_top_left_radius=0,
-                     border_bottom_right_radius=15)
+    pygame.draw.rect(screen, "green", [115, 210, 70, 40], 10, border_radius=15)
+    pygame.draw.rect(
+        screen,
+        "red",
+        [135, 260, 50, 30],
+        0,
+        border_radius=10,
+        border_top_left_radius=0,
+        border_bottom_right_radius=15,
+    )
 
     # Draw an ellipse outline, using a rectangle as the outside boundaries
-    pygame.draw.ellipse(screen, RED, [225, 10, 50, 20], 2) 
+    pygame.draw.ellipse(screen, "red", [225, 10, 50, 20], 2)
 
     # Draw an solid ellipse, using a rectangle as the outside boundaries
-    pygame.draw.ellipse(screen, RED, [300, 10, 50, 20]) 
- 
+    pygame.draw.ellipse(screen, "red", [300, 10, 50, 20])
+
     # This draws a triangle using the polygon command
-    pygame.draw.polygon(screen, BLACK, [[100, 100], [0, 200], [200, 200]], 5)
-  
-    # Draw an arc as part of an ellipse. 
+    pygame.draw.polygon(screen, "black", [[100, 100], [0, 200], [200, 200]], 5)
+
+    # Draw an arc as part of an ellipse.
     # Use radians to determine what angle to draw.
-    pygame.draw.arc(screen, BLACK,[210, 75, 150, 125], 0, pi/2, 2)
-    pygame.draw.arc(screen, GREEN,[210, 75, 150, 125], pi/2, pi, 2)
-    pygame.draw.arc(screen, BLUE, [210, 75, 150, 125], pi,3*pi/2, 2)
-    pygame.draw.arc(screen, RED,  [210, 75, 150, 125], 3*pi/2, 2*pi, 2)
-    
+    pygame.draw.arc(screen, "black", [210, 75, 150, 125], 0, pi / 2, 2)

```

Comment:
For beginners radians may be difficult to understand. I would propose to use `math.radians()` here instead. They can look up radians later,

Reply:
I was thinking about that but I didn't want confusing line breaks, as the line would like be too long then.

Reply:
ok, this sounds reasonable and one way or another they need to learn about radians too :)